orttraining/orttraining/eager/test/ort_ops.py

```diff
@@ -8,6 +8,50 @@
 import numpy as np
 import onnxruntime_pybind11_state as torch_ort
 import torch
+from parameterized import parameterized
+
+# OPS - is a list of  list of [test_operator, the tested_tensor]
+ops = [
+    ["abs", torch.tensor([-1, -2, 3, -6, -7])],
+    ["acos"],
+    ["acosh", torch.rand(6).uniform_(1, 9999)],  # limits[1,INF]
+    ["asinh"],
+    ["atanh", torch.rand(6).uniform_(-1, 1)],
+    ["asin", torch.rand(6).uniform_(-1, 1)],
+    ["atan"],
+    ["ceil"],
+    ["cos"],
+    ["cosh"],
+    ["erf"],
+    ["exp"],
+    ["floor"],
+    # ["isnan",       torch.tensor([1, float('nan'), 2])]]
+    ["log", torch.abs(torch.rand(6)) + 0.5],
+    ["reciprocal"],
+    ["neg", torch.randn(10)],
+    ["round"],
+    ["relu"],
+    # ["selu",        torch.randn(10)]]
+    ["sigmoid"],
+    ["sin"],
+    ["sinh"],
+    ["sqrt", torch.randn(10).uniform_(1, 9999)],
+    ["tan"],
+    ["tanh"],
+]
+# ["nonzero",    torch.tensor([0, 2, 1, 3])]]
+# ["sign",        ]]
+# ["hardsigmoid", ],
+# ["isinf",       ],
+# ["det"          ]]
+
+
+def rename_func_to_op(testcase_func, param_num, param):
+    return f"test_{parameterized.to_safe_name(str(param.args[0]))}"
+
+
+def rename_func_to_inplace(testcase_func, param_num, param):
+    return f"test_{parameterized.to_safe_name(str(param.args[0]))}_"
```

Comment:
thanks for working on this. we can think about generate the test cases from the generation script too.